Code/GraphMol/MolDraw2D/catch_tests.cpp

```diff
@@ -8531,4 +8531,131 @@ M  END)CTAB"_ctab;
+   -1.9872    0.9767    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
+   -2.8547   -0.1585    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -2.1992   -0.6594    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -2.3055   -1.4777    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -3.0669   -1.7949    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -3.7225   -1.2939    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+   -3.6163   -0.4759    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+  2  3  1  0  0  0  0
+  3  4  2  0  0  0  0
+  3  5  1  0  0  0  0
+  5  6  1  0  0  0  0
+  6  7  1  0  0  0  0
+  7  8  1  0  0  0  0
+  8  9  1  0  0  0  0
+  9 10  1  0  0  0  0
+ 10  5  1  0  0  0  0
+M  END
+$MOL
+
+  Mrv1920 09012316112D
+
+ 20 21  0  0  0  0            999 V2000
+    7.3862    1.9786    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    7.2800    1.1605    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
+    7.9356    0.6596    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    8.6971    0.9768    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
+    7.8294   -0.1586    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    8.4850   -0.6595    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    8.3789   -1.4777    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    7.6173   -1.7950    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    6.9619   -1.2939    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    7.0679   -0.4758    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
+    6.4124    0.0252    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    6.5186    0.8432    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
+    5.6509   -0.2920    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
+    4.9954    0.2089    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    4.2337   -0.1084    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    3.5784    0.3926    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    2.8166    0.0753    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    2.7107   -0.7427    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    3.3661   -1.2438    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+    4.1276   -0.9266    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+  2  3  1  0  0  0  0
+  3  4  2  0  0  0  0
+  3  5  1  0  0  0  0
+  5  6  1  0  0  0  0
+  6  7  1  0  0  0  0
+  7  8  1  0  0  0  0
+  8  9  1  0  0  0  0
+  9 10  1  0  0  0  0
+ 10 11  1  0  0  0  0
+ 11 12  2  0  0  0  0
+ 11 13  1  0  0  0  0
+ 13 14  1  0  0  0  0
+ 14 15  1  0  0  0  0
+ 15 16  4  0  0  0  0
+ 16 17  4  0  0  0  0
+ 17 18  4  0  0  0  0
+ 18 19  4  0  0  0  0
+ 19 20  4  0  0  0  0
+ 10  5  1  0  0  0  0
+ 20 15  4  0  0  0  0
+M  END)RXN";
+
+    std::unique_ptr<ChemicalReaction> rxn(RxnBlockToChemicalReaction(rxnBlock));
+    MolDraw2DSVG drawer(-1, -1);
+    auto reactIter = rxn->beginReactantTemplates();
+    REQUIRE_THROWS(MolDraw2DUtils::setACS1996Options(
```

Comment:
If the exception is changed to ValueError, then this should also check specifically for that exception type.